wordpress-mobile/WordPress-Android
# Add Google credential manager support

This is PR supports passkeys using Google Credential Manager. 

This PR only contains hash updates for the Login and FluxC libraries. All the credit goes to @ThomazFB; he added all the support. 🙇 

FluxC https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2949 
Login-Flow-Android https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/135

**Merge Instructions**
- Ensure the two PR above have been merged
- Wait for the hash from the merges
- Update this PR with the proper hashes
- Remove the Not Ready for Merge label
- Merge as normal

-----

## To Test:

Instructions are in the Login-Flow PR https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/135; please use the WP app as the basis for testing and not the Woo app. I tested using a release build of WP (which can be built locally).

Note: You must ensure that Google Password Manager is turned on when testing device passkeys (as in the instructions). On your device, search for Passwords & Accounts, then make sure the switch is on for Google Password Manager. 

-----

## Regression Notes

1. Potential unintended areas of impact
Passkey login does not work

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

-----

## PR Submission Checklist:

-----

## Testing Checklist:
